lib/webapp.js

```diff
@@ -134,6 +149,51 @@ DataserviceContext.prototype = {
   makeErrorObject: zluxUtil.makeErrorObject
 };
 
+function createDataserviceStorage() {
+  return process.clusterManager.getStorage().then( function(clusterStorage) {
+    
+    clusterStorage.getStorage = function () {
+      return process.clusterManager.getStorage().then((storage) => {
+        clusterStorage.storageDict = storage.storageDict;
+        return storage.storageDict;
+      });
+    }
+
+    clusterStorage.setStorageValue = function (id, value) {
+      clusterStorage.storageDict[id] = value;
```

Comment:
This doesn't make sense: it looks like there are 2 objects when there should be 1: the clustermanager's store. what is clusterStorage.storageDict doing?

Reply:
Yep, there are two. My misunderstanding! See my reply.